src/app/screens/Register/index.tsx

```diff
@@ -10,11 +10,12 @@ import { AUTH_FIELDS } from '~constants/fields';
 import { useLazyRequest } from '~app/hooks/useRequest';
 import { apiRegister } from '~services/Auth';
 import AlertMessage from '~components/AlertMessage';
-import { User } from '~utils/types';
+import { User, FunctionForTest } from '~utils/types';
+import { getErrorMessage } from '~utils/getErrorMessageApi';
 
 import styles from './styles.module.scss';
 
-function Register() {
+function Register({ mockFunction }: FunctionForTest) {
```

Comment:
there should not be any testing refered on a component except for `data-testid` or roles, this should not exist